src/python/pants/backend/docker/subsystems/dockerfile_parser_test.py

```diff
@@ -181,7 +187,7 @@ def test_copy_source_references(rule_runner: RuleRunner) -> None:
                 FROM base
                 COPY a b /
                 COPY --option c/d e/f/g /h
-                ADD ignored
+                ADD ignored ignored
```

Comment:
I changed this because `ADD` requires both src and dest, and tree-sitter stopped parsing after this point otherwise.